Source/Core/AudioCommon/OpenALStream.cpp

```diff
@@ -149,10 +155,13 @@ void OpenALStream::SoundLoop()
   memset(uiBuffers, 0, numBuffers * sizeof(ALuint));
   uiSource = 0;
 
-  // Checks if a X-Fi is being used. If it is, disable FLOAT32 support as this sound card has no
-  // support for it even though it reports it does.
+  if (alIsExtensionPresent("AL_EXT_float32"))
+    float32_capable = true;
+
+  // Checks if a X-Fi is being used. If it is, enable FIXED32 support as this sound card
+  // is the only one that supports it for now.
   if (strstr(alGetString(AL_RENDERER), "X-Fi"))
-    float32_capable = false;
+    fixed32_capable = true;
```

Comment:
Oh right, I had forgot Dolphin is 64 bit (never liked the [explanation](https://it.dolphin-emu.org/blog/2014/05/19/obituary-32bit/) for this tbh) :s 

I wonder if with respect to OpenAL soft there's any actual downside then.